File: Said_Coley_Romero/Programas/Dart/Trabajo_29_de_Julio.dart
class Humano{
  String? nombre;
  int? edad;
  int? peso;

}

class Animal{
  String? raza;
  String? color;
  int? peso;

}

class Carro{
  String? color;
  int? peso;
  String? placa;

}

void main(){
  Humano humano = new Humano();
  humano.nombre = "Daniel";
  humano.edad = 22;
  humano.peso = 70;

  Animal animal = new Animal();
  animal.raza = "Gato";
  animal.color = "Naranja";
  animal.peso = 30;

  Carro carro = new Carro();
  carro.color = "Blanco";
  carro.peso = 1600;
  carro.placa = "SJT645";

}

File: Said_Coley_Romero/Programas/Ruby/Trabajo_29_de_Julio.rb
class Humano
    attr_accessor :nombre, :edad, :peso
  end

  class Animal
    attr_accessor :raza, :color, :peso
  end

  class Carro
    attr_accessor :color, :peso, :placa
  end

  humano = Humano.new
  humano.nombre = "Daniel"
  humano.edad = 22
  humano.peso = 70

  animal = Animal.new
  animal.raza = "Gato"
  animal.color = "Naranja"
  animal.peso = 30

  carro = Carro.new
  carro.color = "Blanco"
  carro.peso = 1600
  carro.placa = "SJT645"